modin/experimental/core/execution/native/implementations/hdk_on_native/hdk_worker.py

```diff
@@ -84,10 +84,11 @@ def executeDML(cls, query):
     @classmethod
     def executeRA(cls, query):
         if query.startswith("execute relalg"):
-            ra = query.removeprefix("execute relalg")
+            ra = query[14:]
```

Comment:
Why?

Reply:
This code failed on Python 3.8, because `removeprefix()` added since 3.9.